src/plugins/intel_cpu/src/nodes/executors/jit/eltwise.cpp

```diff
@@ -0,0 +1,608 @@
+#elif defined(OPENVINO_ARCH_RISCV64)
+    const auto isISASupportedByJIT = ov::intel_cpu::riscv64::mayiuse(ov::intel_cpu::riscv64::gv);
+#else
+    const auto isISASupportedByJIT = false;
+#endif
+    // if dim rank is greater than the maximum possible, we should not use JIT execution
+    if (rank > MAX_ELTWISE_DIM_RANK) {
+        return false;
+    }
+
+    if (!isISASupportedByJIT) {
+        return false;
+    }
+
+    const auto algorithm = attrs.data.algo;
+    if (one_of(algorithm,
+               Algorithm::EltwiseLog,
+               Algorithm::EltwiseBitwiseLeftShift,
+               Algorithm::EltwiseBitwiseRightShift)) {
+        return false;
+    }
+
+#if defined(OPENVINO_ARCH_X86_64)
+    return true;
+
+#elif defined(OPENVINO_ARCH_ARM64)
+    if (one_of(algorithm,
+               Algorithm::EltwiseBitwiseAnd,
+               Algorithm::EltwiseBitwiseNot,
+               Algorithm::EltwiseBitwiseOr,
+               Algorithm::EltwiseBitwiseXor)) {
+        return false;
+    }
+
+    std::vector<ov::element::Type> supported_input_precisions = std::vector<ov::element::Type>{ov::element::f16,
+                                                                                               ov::element::f32,
+                                                                                               ov::element::i32,
+                                                                                               ov::element::i8,
+                                                                                               ov::element::u8};
+
+    std::vector<ov::element::Type> supported_output_precisions = supported_input_precisions;
+    if (one_of(algorithm, Algorithm::EltwiseDivide, Algorithm::EltwiseFloor)) {
+        supported_input_precisions = std::vector<ov::element::Type>{ov::element::f16, ov::element::f32};
+    }
+
+    const auto& postOps = attrs.postOps;
+    if (!postOps.empty()) {
+        // Divide and Floor (issue #138629) operations are supported for fp32 and fp16 only.
+        if (const auto* const scaleShiftPostOp = std::any_cast<ScaleShiftPostOp>(&postOps.back())) {
+            if (scaleShiftPostOp->type() == ScaleShiftPostOp::Type::divide) {
+                supported_input_precisions = std::vector<ov::element::Type>{ov::element::f16, ov::element::f32};
+            }
+        } else if (const auto* const activationPostOp = std::any_cast<ActivationPostOp>(&postOps.back())) {
+            if (activationPostOp->type() == ActivationPostOp::Type::floor) {
+                supported_input_precisions = std::vector<ov::element::Type>{ov::element::f16, ov::element::f32};
+            }
+        }
+    } else {
+        supported_output_precisions = supported_input_precisions;
+    }
+
+#elif defined(OPENVINO_ARCH_RISCV64)
+    if (!one_of(algorithm,
+                Algorithm::EltwiseAbs,
+                Algorithm::EltwiseAdd,
+                Algorithm::EltwiseLogicalAnd,
```

Comment:
Please note that there are [missed](https://github.com/openvinotoolkit/openvino/blob/b11ac1cfbf4224ff6c721e0337d58c5fd01436c7/src/plugins/intel_cpu/src/nodes/eltwise.cpp#L876) LogicalXor, LoginalNot, Equal, Erf

Reply:
`Logical` and `Comparison` ops are covered only by shared tests - there are not CPU specific tests which check primitive type (`jit` vs `ref`). So tests for `LogicalXor`, `LogicalNot` and `Equal` won't be failed

As for `Erf`, I merged the PR with JIT Emitter support recently - I believe that if you rebase on the master branch, tests will be failed - `Erf` is covered by CPU specific tests